src/editor/CMakeLists.txt

```diff
@@ -1,5 +1,6 @@
 # Translation files in src/editor/translations (without .ts)
-set(TS dustrac-editor_fi dustrac-editor_it dustrac-editor_cs dustrac-editor_fr dustrac-editor_de dustrac-editor_ru)
+set(TS dustrac-editor_fi dustrac-editor_it dustrac-editor_cs dustrac-editor_fr dustrac-editor_de dustrac-editor_ru 
+dustrac-game_nl)
```

Comment:
`dustrac-game_nl` added here, but should be `dustrac-editor_nl` I guess..

Reply:
Oops, you're right. Fixed now.